scout/parse/variant/clnsig.py

```diff
@@ -76,38 +82,50 @@ def parse_clnsig(variant, transcripts=None):
             for accession, significance, revstat in zip(
                 accessions, significances, revstats
             ):
-                clnsig_accsessions.append(
+                clnsig_accessions.append(
                     {
                         "value": int(significance),
                         "accession": accession,
                         "revstat": revstat,
                     }
                 )
 
-    return clnsig_accsessions
+    return clnsig_accessions
 
 
 def is_pathogenic(variant):
     """Check if a variant has the clinical significance to be loaded
-    
-    We want to load all variants that are in any of the predefined categories regardless of rank 
+
+    We want to load all variants that are in any of the predefined categories regardless of rank
     scores etc.
-    
+
     Args:
         variant(cyvcf2.Variant)
-    
+
     Returns:
         bool: If variant should be loaded based on clinvar or not
     """
+
     load_categories = set(
         [
             "pathogenic",
             "likely_pathogenic",
-            "conflicting_interpretations_of_pathogenecity",
```

Comment:
This typo made it so that all the conflicting interpretation were NOT loaded no matter what before.

Reply:
😑 
Well found anyway!